coreteam.py

```diff
@@ -0,0 +1,71 @@
+##
+# List of PrestaShop employees or subcontractors
+#
+
+CORE_TEAM = (
+    'PierreRambaud',      # Pierre R.
+    'LittleBigDev',       # Bastien B.
+    'Quetzacoalt91',      # Thomas N.
+    'azouz-jribi',        # Azouz Jribi (Sifast)
+    'eternoendless',      # Pablo B.
+    'fatmaBouchekoua',    # Fatma B. (Sifast)
+    'jocel1',             # Jocelyn F.
+    'mickaelandrieu',     # Mickael A.
+    'rGaillard',          # Remi G.
+    'tomlev',             # Thomas L.
+    'toutantic',          # Aurelien
+    'slorenzini',         # Sarah L.
+    'LouiseBonnard',      # Louise B.
+    'ttoine',             # Antoine T.
+    'colinegin',          # Coline G.
+    'marionf',            # Marion F.
+    'matks',              # Matthieu F.
+    'khouloudbelguith',   # Khouloud B. (Sifast)
+    'fouratachour',       # Fourat A.
+    'sLorenzini',         # Sarah L.
+    'apacios',            # Adrien P.
+    'jolelievre',         # Jonathan L.
+    'clementdaubeuf',     # Clement D.
+    'MrBaiame',           # Alexis V.
+    'Joukz',              # Julien G.
+    'david-piatek',       # David P.
+    'yelnatss',           # Marvin S.
+    'CaptainYouz',        # Anas M.
+    'Matt75',             # Matthias R.
+    'SimonGrn',           # Simon G.
+    'mbadrani',           # Mehdi B.
+    'boubkerbribri',      # Boubker B.
+    'matthieu-rolland',   # Matthieu R.
+    'Progi1984',          # Franck L.
```

Comment:
Shouldn't we delete the ones who no longer work in it?

Reply:
Maybe. But if we do this, the generator cannot be used on older PRs.